<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendWidget</class>
 <widget class="QWidget" name="SendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Переименование</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Тема письма</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_subject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Папка с работами участников</string>
   </property>
   <widget class="QPushButton" name="pushButton_personalAttachments">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_personalAttachments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Файл с почтами участников</string>
   </property>
   <widget class="QPushButton" name="pushButton_emailBase">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_emailBase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Папка с приложениями</string>
   </property>
   <widget class="QPushButton" name="pushButton_attachments">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_attachments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Текст письма в формате HTML</string>
   </property>
   <widget class="QPushButton" name="pushButton_emailBody">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_emailBody">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Имена участников</string>
   </property>
   <widget class="QTextEdit" name="textEdit_participants">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_startSend">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать отправку</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Логин и пароль от электорнной почты</string>
   </property>
   <widget class="QPushButton" name="pushButton_emailKey">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_emailKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_upload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Протокол</string>
   </property>
   <widget class="QPushButton" name="pushButton_protocol">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_protocol">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
